curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63b61584def8fa2ebcc259e0.md

```diff
@@ -7,7 +7,17 @@ dashedName: step-19
 
 # --description--
 
-You need to split your `str` into individual characters. Remember that strings have a `.split()` method, and if you pass an empty string it will split on every character.
+You need to split your `str` into individual characters. You can use the <dfn>split()</dfn> method to do this.
+
+The `split()` method splits a string into an array of substrings, and returns the new array. You can pass in an optional separator which tells the computer where each split should happen.
+
+Here is an example of how to use it:
+
+```js
+const str = 'Hello World';
+const strArray = str.split('');
+// ["H", "e", "l", "l", "o", " ", "W", "o", "r", "l", "d"]
+```
 
 Declare a `strArray` variable and assign it that split value.
```

Comment:
I think this instruction is not clear enough.

```suggestion
Split the string passed into the `cleanInputString` function into an array of individual characters and assign it to the `strArray` variable.
```

I'm guessing this could be shortened a little, feel free to do so. My main concern was that the phrase "assign it that split value" was too ambiguous.